src/__tests__/base.test.ts

```diff
@@ -55,36 +42,75 @@ describe("Base", () => {
     })
 
     it("inserts correct version header", async () => {
-      const json = await fetchPath(Service.CAR, "api/path")
+      const json = await fetchPath({ path: "/api/path", options: {} })
 
       expect(json).toEqual({ ok: true })
       expect(fetch).toHaveBeenCalledWith(expect.any(String), {
+        method: "GET",
         headers: expect.objectContaining({
           Accept: `application/vnd.carforyou.v1+json`,
         }),
       })
     })
 
     it("allows setting custom headers", async () => {
-      const json = await fetchPath(Service.CAR, "api/path", {
-        headers: { Foo: "bar" },
+      const json = await fetchPath({
+        path: "api/path",
+        options: {
+          headers: { Foo: "bar" },
+        },
       })
 
       expect(json).toEqual({ ok: true })
       expect(fetch).toHaveBeenCalledWith(expect.any(String), {
+        method: "GET",
         headers: expect.objectContaining({
           Foo: "bar",
         }),
       })
     })
 
+    it("sets recaptcha header if token is provided", async () => {
+      const json = await fetchPath({
+        path: "/api/path",
+        options: { recaptchaToken: "token" },
+      })
+
+      expect(json).toEqual({ ok: true })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.any(String),
+        expect.objectContaining({
+          headers: {
+            Accept: "application/vnd.carforyou.v1+json",
+            "Content-Type": "application/json",
+            "Recaptcha-Token": "token",
+          },
+        })
+      )
+    })
+
+    it("passes fetch options to fetch", async () => {
+      const json = await fetchPath({
+        path: "/api/path",
+        options: { mode: "no-cors" },
```

Comment:
Do we really need that we only make use of it in this test case but not in the code itself...

Reply:
I recall using cors modes for some debugging

Reply:
Ohh you just typed that internally. Messed that up... nvm